src/kvirc/kernel/KviIrcConnectionServerInfo.cpp

```diff
@@ -275,6 +273,566 @@ KviBasicIrcServerInfo::~KviBasicIrcServerInfo()
+	}
+	return KviHybridServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviBahamutIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'A': return __tr2qs("A: Server administrator"); break;
+		case 'C': return __tr2qs("C: Only accept private messages from users who share common channels"); break;
+		case 'F': return __tr2qs("F: Can bypass the ircd's recvq throttling"); break;
+		case 'I': return __tr2qs("I: Hide oper status"); break;
+		case 'K': return __tr2qs("K: Can see U:lined kill messages"); break;
+		case 'R': return __tr2qs("R: Receive private messages from registered nicks only"); break;
+		case 'S': return __tr2qs("S: Connected over SSL"); break;
+		case 'X': return __tr2qs("X: Squelch without notice"); break;
+		case 'a': return __tr2qs("a: Services administrator"); break;
+		case 'b': return __tr2qs("b: Can see CHATOPS notices"); break;
+		case 'e': return __tr2qs("e: Recipient for blocked DCC notices"); break;
+		case 'f': return __tr2qs("f: Recipient for flood warnings"); break;
+		case 'g': return __tr2qs("g: Recipient for GLOBOPS notices"); break;
+		case 'h': return __tr2qs("h: Available for help (HelpOp)"); break;
+		case 'j': return __tr2qs("j: Recipient for rejected client notices"); break;
+		case 'm': return __tr2qs("m: Recipient for spambot notices"); break;
+		case 'n': return __tr2qs("n: Recipient for routing notices"); break;
+		case 'r': return __tr2qs("r: Registered"); break;
+		case 's': return __tr2qs("s: Recipient for server KILL messages"); break;
+		case 'x': return __tr2qs("x: Squelch with notice"); break;
+		case 'y': return __tr2qs("y: Can see certain information requests (e.g. /stats)"); break;
+	}
+	return KviBasicIrcServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviUnreal32IrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'A': return __tr2qs("A: Server administrator"); break;
+		case 'B': return __tr2qs("B: Marks as being a bot"); break;
+		case 'C': return __tr2qs("C: Co-Admin"); break;
+		case 'F': return __tr2qs("F: Override filter settings"); break;
+		case 'G': return __tr2qs("G: Filter out bad words"); break;
+		case 'H': return __tr2qs("H: Hide oper status"); break;
+		case 'I': return __tr2qs("I: Idle time hidden from WHOIS"); break;
+		case 'N': return __tr2qs("N: Network administrator"); break;
+		case 'R': return __tr2qs("R: Receive private messages from registered nicks only"); break;
+		case 'S': return __tr2qs("S: Network service"); break;
+		case 'T': return __tr2qs("T: Prevent CTCPs"); break;
+		case 'V': return __tr2qs("V: Connected over WebTV"); break;
+		case 'W': return __tr2qs("W: Recipient for notices on WHOIS"); break;
+		case 'a': return __tr2qs("a: Services administrator"); break;
+		case 'd': return __tr2qs("d: Deaf"); break;
+		case 'g': return __tr2qs("g: Can send & read LOCOPS and GLOBOPS"); break;
+		case 'h': return __tr2qs("h: Available for help (HelpOp)"); break;
```

Comment:
I'm going to just use the lowercase `helpop` string here, since we have `locop` and `oper` and they all carry the same definition structure, and that is also how IRCds refer to them as well (although unreal uses `HelpOp` but for the sake of consistency, I'm lowercasing it since it's not a command.)